core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -1805,38 +1804,24 @@ class KafkaApis(val requestChannel: RequestChannel,
   def handleLeaveGroupRequest(request: RequestChannel.Request): Unit = {
     val leaveGroupRequest = request.body[LeaveGroupRequest]
 
-    val members = leaveGroupRequest.members.asScala.toList
-
-    if (!authHelper.authorize(request.context, READ, GROUP, leaveGroupRequest.data.groupId)) {
+    def sendResponse(response: AbstractResponse): Unit = {
       requestHelper.sendResponseMaybeThrottle(request, requestThrottleMs => {
-        new LeaveGroupResponse(new LeaveGroupResponseData()
```

Comment:
In some of the other PRs we had trace logs. Did we want to do that here?

Reply:
Added it.